frontend/appflowy_flutter/lib/workspace/presentation/home/menu/sidebar/sidebar.dart

```diff
@@ -46,33 +45,27 @@ class HomeSideBar extends StatelessWidget {
           create: (_) => FavoriteBloc()..add(const FavoriteEvent.initial()),
         )
       ],
-      child: BlocListener<MenuBloc, MenuState>(
```

Comment:
Why delete this line? It will cause the CI to fail.

Reply:
I removed it in good faith.

There's a mismatch between MenuBloc and ViewBloc, I'll resolve it.